updating the path to producing the usage report (#6951)
Particular/docs.particular.net: nservicebus/usage/index.md

  - nservicebus/throughput-tool
---

## Using ServicePulse to measure usage

ServicePulse will generate a usage report containing NServiceBus endpoint count and throughput summary. The report is saved as a file onto the local machine, which needs to be sent to Particular upon request, usually at license renewal time.

> ServicePulse provides the user with;

Reviewer: ```suggestion
ServicePulse provides the user with:
```

- The ability to [view system usage](/servicepulse/usage.md#viewing-usage-summary) at any time
- The ability to [specify if a detected queue should not be included in license pricing](/servicepulse/usage.md#setting-an-endpoint-type)
- Improved endpoint detection for endpoints that have Monitoring or Audit enabled
- Once setup, there's no more work required, just a button press once a year to generate the usage report

The Usage page allows for viewing the endpoint usage summary of a system using NServiceBus.
A usage report can be downloaded here at license renewal time.

> [!NOTE]
> The usage data collection functionality requires ServicePulse version 1.40 or later, and ServiceControl version 5.4 or later. Additionally, if using RabbitMQ broker, version 3.10.0 or higher is required.

> [!NOTE]
> While customers are not currently required to use ServicePulse, starting 1 January 2026, it will become mandatory for all projects in active development. If you have any issues installing ServicePulse in your environment, please reach out so we can help find a path forward.

> Customers who are not able to use ServicePulse should [contact us](https://particular.net/contact)

Author: ```suggestion
Customers who are not able to use ServicePulse can either contact their Account Manager or [Contact our Customer Success Team](https://particular.net/contact)
```

  - nservicebus/throughput-tool
---

## Using ServicePulse to measure usage

ServicePulse will generate a usage report containing NServiceBus endpoint count and throughput summary. The report is saved as a file onto the local machine, which needs to be sent to Particular upon request, usually at license renewal time.

> ServicePulse provides the;

Author: ```suggestion
ServicePulse provides the user with;
```

> - ability to [view system usage](/servicepulse/usage.md#viewing-usage-summary) at any time

Author: ```suggestion
- The ability to [view system usage](/servicepulse/usage.md#viewing-usage-summary) at any time
```

> - ability to [specify if a detected queue should not be included in license pricing](/servicepulse/usage.md#setting-an-endpoint-type)

Author: ```suggestion
- The ability to [specify if a detected queue should not be included in license pricing](/servicepulse/usage.md#setting-an-endpoint-type)
```

> - incorporated into the existing Particular Platform - [the report can be generated directly from ServicePulse](/servicepulse/usage.md#download-a-usage-report)

Author: ```suggestion

```

> - improved endpoint detection for endpoints that have Monitoring or Audit enabled

Author: ```suggestion
- Improved endpoint detection for endpoints that have Monitoring or Audit enabled
```

> - once setup, there's no more work required, just a button press once a year to generate the usage report

Author: ```suggestion
- Once setup, there's no more work required, just a button press once a year to generate the usage report
```
